controllers/cluster.weave.works/existinginframachine_controller.go

```diff
@@ -1318,10 +1326,27 @@ func (m MachineMapper) Map(mo handler.MapObject) []reconcile.Request {
 			log.Errorf("Couldn't find node matching machine: %s", machine.Name)
 			continue
 		}
+
+		isOrig, err := m.reconciler.isOriginalMaster(ctx, node)
+		if err != nil {
+			log.Errorf("Could not determine if node is original master: %s", node.Name)
+			continue
```

Comment:
Can you elaborate on the reason to continue here?  Is it possible it could be a master but fail the check for the original master?  Is there harm in adding it to the requeue list of workers?